Wabbajack.Common/Utils.cs

```diff
@@ -27,10 +27,17 @@ namespace Wabbajack.Common
 {
     public static class Utils
     {
-        public static bool IsMO2Running(string mo2Path)
```

Comment:
I like this refactor.  More generic so we can use it for lots of purposes.
Someone can readd an explicit IsMO2Running() /w the parameters hardcoded if they really want to, or we can have the "ModOrganizer" string a static const somewhere for easy reference when using this function